app/controllers/users_controller.rb

```diff
@@ -1,14 +1,16 @@
-class UsersController < Clearance::UsersController
+class UsersController < ApplicationController
+  before_action :redirect_signed_in_users
```

Comment:
`before_action :redirect_to_root, if: :signed_in?`

Reply:
Is it worth to introduce this method at `ApplicationController` level? It is currently missing.